CONTRIBUTING.md

```diff
@@ -261,6 +262,31 @@ alias lt='lr test'
 alias lw='lr watch'
 ```
 
+### eslint
+
+The CDK uses eslint to 'lint' all ts and js files in the CDK repo. It is executed automatically as part of `yarn build`.
+
+To run `eslint` on a specific package, at the root of the package, run
+
+```bash
+eslint . --ext=.ts,.js
+```
+
+All packages in the repo use a standard base configuration found at [eslintrc.js](tools/cdk-build-tools/config/eslintrc.js).
+This can be customized for any package by modifying the `.eslintrc` file found at its root.
+
+If you're using the VS Code and would like to see eslint violations on it, install the [eslint
+extension](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) and add the following to the
+[settings](https://code.visualstudio.com/docs/getstarted/settings).
+
+```json
+"eslint.workingDirectories": [
+  { "pattern": "./packages/@aws-cdk/*" },
```

Comment:
Can’t we do `packages/*`?

Reply:
Unfortunately not. It has to be these 3 separate statements, for it to work on all packages.